dashboard/test/ui/features/applab/eyes.feature

```diff
@@ -1,7 +1,7 @@
 @eyes
 @dashboard_db_access
 @as_student
-Feature: App Lab Eyes
+Feature: App Lab Eyes -  Part 1
```

Comment:
Could we call this file eyes1 so that we can find it by grepping for applab_eyes1 (rather than applab_eyes which is more ambiguous)?